tests/src/test/java/ca/bc/gov/open/jag/efiling/services/FilingPackageService.java

```diff
@@ -12,6 +12,22 @@
     @Value("${EFILING_HOST:http://localhost:8080}")
     private String eFilingHost;
 
+    public Response getFilingPackages(String accessToken, String parentApp) {
+
+        RequestSpecification request = RestAssured
+                .given()
+                .auth()
+                .preemptive()
+                .oauth2(accessToken);
+
+        return request.queryParam("parentApplication", parentApp)
+                .when()
+                .get(MessageFormat.format("{0}/filingpackages", eFilingHost))

```

Comment:
Only a suggestion. But the paths could be moved to keys.

Reply:
Thanks @TayGov, great idea. As it's the same on other services as well. Will do it as a refactor.